src/commands.ts

```diff
@@ -568,7 +568,18 @@ export function setCellAttr(name: string, value: unknown): Command {
   return function (state, dispatch) {
     if (!isInTable(state)) return false;
     const $cell = selectionCell(state);
-    if ($cell.nodeAfter!.attrs[name] === value) return false;
+
+    const isMultipleCellSelection =
+      state.selection instanceof CellSelection &&
+      state.selection.isMultipleCellSelection();
+
+    // If selection is multiple cell selection, don't bail early
+    // this allows to set the attribute to the same value for all cells, even
+    // if some cells already have the attribute set to that value
+    if (!isMultipleCellSelection && $cell.nodeAfter!.attrs[name] === value) {
```

Comment:
Hmm, don't we want to go through each of the cells and bail if all of them have the attribute? For example, we shouldn't be able to clear the color of a selection of cells that all have no color. I think this version is okay if that's hard or fiddly though.